### PR TITLE
Suggested edits

### PR DESCRIPTION
**Pull request by magnurud**

### CODE REVIEW

File: docs/concepts/training-data.mdx [hunk 1]
```diff
@@ -5,19 +5,77 @@ sidebar_position: 4
...
-A _data bundle_ in Cradl is a model-specific collection of one or more [Datasets](datasets.mdx). Before training a model, you must specify a data bundle that the model will use as training data. Since the quality of the trained model depends on the quality of the training data, we require that any data bundles used for training must pass a statistical test in the form of a [Data Report](training-data.mdx#data-report) before training can begin.
+A _training_ in Cradl is the process of training a machine learning model on the data you provide.
+It can be summed up in 3 simple steps:
+1. User requests training in the Cradl app, or with one of our SDKs or CLI.
+2. Our team of Data Scientists verify that the training is feasible, and starts the training.
+3. User receives a new model that has been trained according to the request.
 
-## Creating a data bundle
+:::info
+We make sure to always provide a model whose architecture is based on the latest research to make sure that our customers get the most out of their data.
```
Comment (augustak) [inline docs/concepts/training-data.mdx:17]: Does it sound like we are not developing our own models?

File: docs/concepts/training-data.mdx [hunk 2]
```diff
@@ -5,19 +5,77 @@ sidebar_position: 4
...
+<TabItem value="python" label="Python">
+
+```python
+data_bundle = client.create_training(model_id='<modelId>', data_bundle_ids=['<dataBundleId>'], name='Initial training of receipt model')
+```
+
+</TabItem>
+</Tabs>
+
+After the training is requested you will receive an email that verifies the request, and when the training is completed.
```
Comment (augustak) [inline docs/concepts/training-data.mdx:58]:
```suggestion
You will be notified on email when a training is requested or changes status.
```

File: docs/concepts/training-data.mdx [hunk 3]
```diff
@@ -5,19 +5,77 @@ sidebar_position: 4
...
+You can also check the request of your training, by looking at its status.
```
Comment (augustak) [inline docs/concepts/training-data.mdx:59]:
```suggestion
You can also check the status of your training by getting the training object with the CLI or in the Web app.
```

File: docs/concepts/training-data.mdx [hunk 4]
```diff
@@ -5,19 +5,77 @@ sidebar_position: 4
...
+
+
+## Data Bundle
+
+A _data bundle_ in Cradl is a model-specific collection of one or more [Datasets](datasets.mdx).
+Before training a model, you must specify a data bundle that the model will use as training data.
+Since the quality of the trained model depends on the quality of the training data,
+we require that any data bundles used for training must pass a statistical test in the form of a [Data Report](training-data.mdx#data-report) before training can begin.
```
Comment (augustak) [inline docs/concepts/training-data.mdx:67]: What are the requirements for a statistical test to pass?

**Review by augustak — CHANGES_REQUESTED** [review]

File: docs/concepts/training-data.mdx (hunk 3 above)
Comment (magnurud) [reply docs/concepts/training-data.mdx:59]:
```suggestion
You can also check the status of your training by inspecting the training object with the CLI or in the Cradl app.
```

File: docs/concepts/training-data.mdx (hunk 4 above)
Comment (magnurud) [reply docs/concepts/training-data.mdx:67]: They are often subject to change, so putting down something specific here would quickly be outdated I am afraid.

File: docs/concepts/training-data.mdx (hunk 1 above)
Comment (magnurud) [reply docs/concepts/training-data.mdx:17]:
```suggestion
We develop models whose architecture is based on the latest research to make sure that our customers get the most out of their data.
```

Comment (augustak) [reply docs/concepts/training-data.mdx:17]: @staalezh Any comments?

Comment (staalezh) [reply docs/concepts/training-data.mdx:17]: Is this relevant for understanding the training process? Maybe we can elaborate on this elsewhere, e.g. in a separate technology section?

Comment (magnurud) [reply docs/concepts/training-data.mdx:17]: A valid point, I was thinking a little promo wouldn't harm, but if no one else agree, I am open to omit it.

File: docs/concepts/training-data.mdx [hunk 5]
```diff
@@ -5,19 +5,77 @@ sidebar_position: 4
...
+</TabItem>
+</Tabs>
+
+You will be notified on email when a training is requested or changes status.
+You can also check the status of your training by inspecting the training object with the CLI or in the Cradl app.
+
+
+## Data Bundle
+
+A _data bundle_ in Cradl is a model-specific collection of one or more [Datasets](datasets.mdx).
```
Comment (staalezh) [inline docs/concepts/training-data.mdx:64]:
```suggestion
A _data bundle_ is a model-specific collection of one or more [Datasets](datasets.mdx).
```

File: docs/concepts/training-data.mdx [hunk 6]
```diff
@@ -5,19 +5,77 @@ sidebar_position: 4
...
+Before training a model, you must specify a data bundle that the model will use as training data.
+Since the quality of the trained model depends on the quality of the data,
+we will make a [Data Report](training-data.mdx#data-report) that needs to be approved by one of our Data Scientists before the training can begin.
```
Comment (staalezh) [inline docs/concepts/training-data.mdx:67]:
```suggestion
Before the training job is started, one of our data scientists will review the auto-generated [Data Report](training-data.mdx#data-report) as part of our QA process.
```

File: docs/get-started/train-model.mdx [hunk 7]
```diff
@@ -11,7 +11,13 @@ import TabItem from '@theme/TabItem';
 
 We have now defined a model and created a dataset. The next step is to train the model, but first we need to tell it what data to use when training. At this point, it would also be wise to take a step back and make sure that our training data is of good quality, so that the outcome of the training will be satisfactory. Good quality training data is varied and has lots of examples for all the fields we want the model to extract.
 
-To tell the model which data to use, we select datasets containing our training data and attach them to the model in the form of a [data bundle](../concepts/training-data.mdx). Doing so will automatically create a [data quality report](../concepts/training-data.mdx#data-report) on the training data.
+:::tip
+
+The simplest way to request a training is to use the training wizard in the Cradl app, just go to [**Models**](https://app.cradl.ai/models) > **Your model** > **Training jobs** > **New training**.
```
Comment (staalezh) [inline docs/get-started/train-model.mdx:16]:
```suggestion
The simplest way to start a training is to use the training wizard in the Cradl app, just go to [**Models**](https://app.cradl.ai/models) > **Your model** > **Training jobs** > **New training**.
```

File: docs/concepts/training-data.mdx [hunk 8]
```diff
@@ -140,11 +198,11 @@ While the number of documents in total is good, we would not be able to guarante
 
 #### Good variation
 
-Machine learning algorithms are impressionable by nature, and can skew their predictions if the training data is skewed. One example of skewness is lack of variation: having too many similar examples in the training data. The variation of your data is quantified in the Variation statistic of the [Data bundle](training-data.mdx#data-report). 
+Machine learning algorithms are impressionable by nature, and can skew their predictions if the training data is skewed. One example of skewness is lack of variation: having too many similar examples in the training data. The variation of your data is quantified in the Variation statistic of the [Data bundle](training-data.mdx#data-report).
```
[the remark this answers is not in the document]
Comment (magnurud) [reply docs/concepts/training-data.mdx:200]:
```suggestion
Machine learning algorithms are adaptable by design, and will learn patterns from the training data. One example of this is if the training data has a large number of similar training examples, the model may be encouraged to prioritize these documents at the expense of the less frequent ones. The variation of your data is quantified in the variation statistic of the [Data bundle](training-data.mdx#data-report).
```

**Review by augustak — APPROVED** [review]

**Pull request merged**